megamek/src/megamek/client/ui/swing/boardview/BoardView1.java

```diff
@@ -2748,13 +2749,6 @@ private void drawHex(Coords c, Graphics boardGraph,
                 if (!useIsometric()) {
                     g.drawImage(scaledImage, 0, 0, this);
                 }
-                // draw a shadow for bridge hex.
-                if (useIsometric()
```

Comment:
No more bridge shadows? :(

Reply:
They're still there! Just a few lines above that.